variables.tf

```diff
@@ -187,6 +181,25 @@ variable "tags" {
   default     = {}
 }
 
+variable "postinstall_script" {
+  default     = <<-EOD
+  #!/bin/bash
+
+  echo 'A post-install script was not provided.'
+  EOD
```

Comment:
so this isn't an explicit suggestion or a blocking question, but do we as a team have an opinionated stance on heredoc format over using file() for non templated things, this script is small enough to not run into any issues, but to align expectations I'm wondering if we want to put all these in the `files/` dir as like postinstall.default.sh

Reply:
I think that sounds reasonable!

Reply:
How does https://github.com/hashicorp/terraform-aws-terraform-enterprise/pull/79/commits/b87a7fc48195f3d15e3b200c313c8dab951e1f11 look?

Reply:
Looks great!